openedx/adg/lms/webinars/admin.py

```diff
@@ -71,52 +75,64 @@ def get_fields(self, request, obj=None):
         """
         fields = super().get_fields(request, obj)
         if not obj:
-            fields.remove('send_update_emails_to_registrants')
+            fields.remove(SEND_UPDATE_EMAILS_FIELD)
         return fields
 
     def save_related(self, request, form, formsets, change):
         """
         Extension of save_related for webinar to send emails when object is created or modified.
         """
+        new_members = []
+        removed_members = []
         if change and any(field in form.changed_data for field in ['co_hosts', 'presenter', 'panelists']):
-            update_webinar_team_registrations(form)
+            new_members, removed_members = get_newly_added_and_removed_team_members(form)
```

Comment:
Can we remove this `if` and move the following in the second `if change` at line 94?
```
if any(field in form.changed_data for field in ['co_hosts', 'presenter', 'panelists']):
        new_members, removed_members = get_newly_added_and_removed_team_members(form)
```

Reply:
I exactly did that, and it cost me many hours 😅. Actually, `super().save_related(request, form, formsets, change)` has to be called after `get_newly_added_and_removed_team_members` for `get_newly_added_and_removed_team_members` to work properly. And we cannot move the super call to the end of the function as it has to be called before line 124 `webinar_team_emails = webinar_emails_for_panelists_co_hosts_and_presenter(webinar)` in the else clause. We want to make the super call as earlies as possible, hence placing it immediately after `get_newly_added_and_removed_team_members`.

Reply:
Oh, thanks for the clarification bro! 
In that case, I don't think we can make many changes to the `save_related` method 😞

Reply:
No worries!